index.js

```diff
@@ -73,7 +74,7 @@ function adjustFilepath (filepath, sourceRoot) {
 }
 
 function instrument (originalCode, filepath, options) {
-    var jsAst = esprima.parse(originalCode, {tolerant: true, loc: true});
+    var jsAst = acorn.parse(originalCode, {locations: true, plugins: {asyncawait: true}});
```

Comment:
Just for curiosity, is acorn work with `asyncawait: true` without `ecmaVersion: 7` ?